src/components/admin/filterNumberRange.js

```diff
@@ -0,0 +1,24 @@
+adminApp.filterNumberRange = {
+  view: function(ctrl, args){
+    return m('.w-col.w-col-3.w-col-small-6', [
+            m('label.fontsize-smaller[for="field-6"]', args.label),
```

Comment:
We need an ID to use the ```for``` attribute in the label, correct?
The problem is that using a generic component we can't hardcode it, so we may have to generate an ID using arguments.